content/docs/guides/branching-github-actions.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Automate branching with GitHub Actions
+subtitle: Create and delete branches with GitHub Actions
+enableTableOfContents: true
+---
+
+Neon provides the following GitHub Actions for working with Neon branches, which you can add to your CI workflows:
+
+- [Create branch action](#create-branch-action)
+- [Delete branch action](#delete-branch-action)
+
+## Create branch action
+
+This GitHub Action creates a new branch in your Neon project.
+
+<Admonition type="info">
+The source code for this action is available on [GitHub](https://github.com/neondatabase/create-branch-action).
+</Admonition>
+
+### Prerequisites
+
+- Using the action requires a Neon API key. For information about obtaining an API key, see [Create an API key](/docs/manage/api-keys#create-an-api-key).
+- Add your Neon API key to your GitHub Secrets:
+    1. In your GitHub repository, go to **Settings** and locate **Secrets** at the bottom of the left sidebar.
+    2. Click **Actions** > **New Repository Secret**.
+    3. Name the secret `NEON_API_KEY` and paste your API key in the **Secret** field
+    4. Click **Add Secret**.
+
+### Example
+
+The following example creates a branch from the `main` branch in your Neon project when a pull request is opened.
```

Comment:
```suggestion
The following example creates a branch from the `main` branch in your Neon project 
```
We can either update the code snippet or remove the "when a pul request is opened" part